src/masonite/api/authentication/AuthenticatesTokens.py

```diff
@@ -0,0 +1,12 @@
+import jwt
+
+
+class AuthenticatesTokens:
+
+    __TOKEN_COLUMN__ = "api_token"
```

Comment:
So we should add this column in the default User model ? 
What would be the type/length of the field ?

Reply:
@girardinsamuel good question. Would probably just be a varchar 255

Reply:
fine to me !